examples/server/demo.cc

```diff
@@ -72,13 +87,11 @@ int main(int, char **) {
     while (std::cin.good()) {
       std::unique_ptr<AlohalyticsBaseEvent> ptr;
       ar(ptr);
-      bricks::rtti::RuntimeDispatcher<AlohalyticsBaseEvent,
-                                      AlohalyticsKeyPairsEvent,
-                                      AlohalyticsIdEvent,
-                                      AlohalyticsKeyValueEvent,
+      bricks::rtti::RuntimeDispatcher<AlohalyticsBaseEvent, AlohalyticsKeyPairsLocationEvent,
+                                      AlohalyticsKeyPairsEvent, AlohalyticsIdEvent, AlohalyticsKeyValueEvent,
                                       AlohalyticsKeyEvent>::DispatchCall(*ptr, processor);
     }
-  } catch (const cereal::Exception & ex) {
+  } catch (const cereal::Exception &ex) {
```

Comment:
True but not critical. I used ex.what() on the server side to check broken cereal packets.